test/unit/mock/services/post.js

```diff
@@ -38,18 +38,11 @@ module.exports = [function () {
,"geometries": [{"type": "Point","coordinates": [-81.707703,30.063236]}]},"properties": {"title": "Wood","description": "333743","marker-color": "#E69327","id": 2448,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2448"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [-81.702675,30.060614]}]},"properties": {"title": "Wood","description": "172534","marker-color": "#E69327","id": 2449,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2449"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [-81.707703,30.063236]}]},"properties": {"title": "Masonry","description": "785275","marker-color": "#E69327","id": 2450,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2450"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [-81.713882,30.102226]}]},"properties": {"title": "Reinforced Concrete","description": "995932","marker-color": "#E69327","id": 2451,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2451"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [-81.711777,30.102261]}]},"properties": {"title": "Masonry","description": "119736","marker-color": "#E69327","id": 2427,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2427"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [-81.702675,30.060614]}]},"properties": {"title": "Wood","description": "172534","marker-color": "#E69327","id": 2431,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2431"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [-81.707703,30.063236]}]},"properties": {"title": "Masonry","description": "785275","marker-color": "#E69327","id": 2432,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2432"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [-81.713882,30.102226]}]},"properties": {"title": "Reinforced Concrete","description": "995932","marker-color": "#E69327","id": 2433,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2433"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [-122.330062,47.603832]}]},"properties": {"title": "jess testing radio button fix","description": "radio button pattern should match pattern library","marker-color": "#A51A1A","id": 2287,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/2287"}},{"type": "Feature","geometry": {"type": "GeometryCollection","geometries": [{"type": "Point","coordinates": [36.817245,-1.283253]}]},"properties": {"title": "Sample post type from Angie","description": "This is my sample image post","marker-color": "#2274B4","id": 24,"url": "http:\/\/qa.api.ushahididev.com\/api\/v3\/posts\/24"}}]})
```

Comment:
It would be nice to be able to use fixtures more in our tests, that way keeping data inline with the backend will be easier.